tests/test_hodgkin.py

```diff
@@ -276,34 +276,40 @@ def test_has_ontology_annotation(self, model):
     def test_get_equations_for(self, graph, model):
         # Get equations for membrane_fast_sodium_current both ordered and unordered
         membrane_fast_sodium_current = model.get_symbol_by_ontology_term(OXMETA, "membrane_fast_sodium_current")
-        equations = model.get_equations_for([membrane_fast_sodium_current])
-        unordered_equations = model.get_equations_for([membrane_fast_sodium_current], False)
-        # There should be 4 in this model
-        assert len(equations) == len(unordered_equations) == 4
+        membrane_stimulus_current_offset = model.get_symbol_by_ontology_term(OXMETA, "membrane_stimulus_current_offset")
+        membrane_stimulus_current_period = model.get_symbol_by_ontology_term(OXMETA, "membrane_stimulus_current_period")
+
+        equations = model.get_equations_for([membrane_fast_sodium_current, membrane_stimulus_current_offset])
+        input_ordered_equations = model.get_equations_for(
+            [membrane_fast_sodium_current, membrane_stimulus_current_offset,
```

Comment:
should membrane_stimulus_current_offset be here twice

Reply:
it's there twice to check that this doesn't lead to duplication of equations even if the input list perhaps contains duplicates.
not 100% sure it's necessary to be fair.

Reply:
If there is a possibility that duplication in the input list would cause duplication of equations; then this is really a separate test. I would do the call with no duplication as well and also check list with duplication produces the correct result

Reply:
ok added.